src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,54 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_STORAGE_SIZE = 10;
+    private K[] keys;
+    private V[] values;
+    private int index = 0;
```

Comment:
```suggestion
    private int index;
```
0 is the default value of int